src/state/application/hooks.tsx

```diff
@@ -107,3 +110,27 @@ export function useNotificationPopup() {
     })
   }, [])
 }
+
+export function useStacklyPopup(): () => Id {
+  const isMobileMedia = useIsMobileByMedia()
+  const [, setClosedStacklyPopup] = useUpdateClosedStacklyPopup()
+  const isYellow = process.env.REACT_APP_STACKLY_POPUP_COLOR === 'yellow'
```

Comment:
Instead of using isYellow can we use `REACT_APP_STACKLY_DESIGN_VERSION` with value `old` or `v1` its confusing why the value `yellow` 

```suggestion
  const isYellow = process.env.REACT_APP_STACKLY_POPUP_COLOR === 'yellow'
```

Reply:
Agree, that's a better naming! Used `REACT_APP_STACKLY_DESIGN_VERSION` with old by default. We can remove this env in the future so it's always the green one showing